tests/load-tests/load-tests-example.json

```
@@ -1,24 +1,32 @@
 {
-  "timestamp": "2023-05-26T13:25:32+02:00",
-  "endTimestamp": "2023-05-26T13:29:58+02:00",
-  "machineName": "orion",
-  "binaryDetails": "Built with go1.19.5 for linux/amd64",
-  "componentRepoUrl": "https://github.com/devfile-samples/devfile-sample-code-with-quarkus",
-  "threads": 5,
-  "usersPerThread": 2,
-  "totalUsers": 10,
+  "timestamp": "2023-07-13T19:23:45+03:00",
+  "endTimestamp": "2023-07-13T19:42:43+03:00",
+  "machineName": "nshprai",
+  "binaryDetails": "Built with go1.20.5 for linux/amd64",
+  "componentRepoUrl": "https://github.com/naftalysh-org/devfile-sample-code-with-quarkus",
+  "threads": 3,
+  "usersPerThread": 10,
+  "totalUsers": 30,
   "pipelineSkipInitialChecks": true,
   "status": "Completed",
-  "createUserTimeAvg": 14.9243592609,
-  "createResourcesTimeAvg": 9.0239237441,
-  "runPipelineSucceededTimeAvg": 205.125,
-  "runPipelineFailedTimeAvg": 145,
+  "createUserTimeAvg": 9.2679308157,
+  "createUserTimeMax": 11.983495356,
+  "createResourcesTimeAvg": 8.550075903933333,
+  "createResourcesTimeMax": 12.397832976,
+  "runPipelineSucceededTimeAvg": 269.6,
+  "runPipelineSucceededTimeMax": 471,
+  "runPipelineFailedTimeAvg": 0,
+  "DeploymentSucceededTimeAvg": 4.733333333333333,
+  "DeploymentSucceededTimeMax": 7,
+  "DeploymentFailedTimeAvg": 0,
   "createUserFailures": 0,
   "createUserFailureRate": 0,
   "createResourcesFailures": 0,
   "createResourcesFailureRate": 0,
   "runPipelineFailures": 2,
   "runPipelineFailureRate": 0.2,
+  "DeploymentFailures": 0,
+  "DeploymentFailureRate": 0,
```

```suggestion
  "deploymentFailures": 0,
  "deploymentFailureRate": 0,
```
Please follow the naming convention here to keep it consistent.

---

Updated